packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -716,14 +774,13 @@ class AccountInternal extends PureComponent {
 
     // Optimistic UI: update the transaction list before sending the data to the database
     this.setState({
-      transactions: [...this.state.transactions, ...reconciliationTransactions],
+      transactions: [...reconciliationTransactions, ...this.state.transactions],
     });
 
     // sync the reconciliation transaction
     await send('transactions-batch-update', {
       added: reconciliationTransactions,
     });
-    await this.refetchTransactions();
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why?

Reply:
Took this out at some point when it was being handled elsewhere and forgot to add it back when I refactored. I'll add it back.